src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -385,6 +386,7 @@ protected override void Free()
             _analyzedNullabilityMapOpt = analyzedNullabilityMapOpt;
             _returnTypesOpt = returnTypesOpt;
             _snapshotBuilderOpt = snapshotBuilderOpt;
+            _localFuncVarUsages = localFuncVarUsagesOpt;
```

Comment:
This is another reason I think we should consider keeping this variable `private`. At a glance it's not clear you're initializing base type behavior here and hence impacting it's ability to do delay creation of items. Think this would be more clear if it was chained into a `base` call on the ctor.